app/models/super_good/solidus_taxjar/order_transaction.rb

```diff
@@ -4,6 +4,10 @@ class OrderTransaction < ActiveRecord::Base
       belongs_to :order, class_name: "Spree::Order"
 
       validates_presence_of :transaction_id
+
+      def self.latest_for(order)
+        where(order: order).order(created_at: :desc).limit(1).first
```

Comment:
We are later going to need to backfill transactions... I could see this being incompatble with this method implementation, as we will create very old transactions in the backfill
 
https://developers.taxjar.com/integrations/sales-tax-reporting/

I think we might wanna record the `transaction_date` on our order_transaction model and sort by that instead.

Reply:
#115 is up which adds the order transaction! We can rebase this on top of that and use the `transaction_date`

Reply:
This change is now rebased over #115 and we have updated the ordering as well as are now setting the `transaction_date` based on the response from Taxjar.